package.json

```diff
@@ -3,6 +3,11 @@
   "version": "1.1.1",
   "description": "Swipe listener for react",
   "main": "lib/Swipe.js",
+  "scripts": {
+      "build": "babel src -d lib",
+      "prepare": "npm run build",
```

Comment:
Ensures correct operation into the future - https://docs.npmjs.com/misc/scripts#prepublish-and-prepare